index.js

```diff
@@ -7,24 +7,24 @@ const tooLowMessage = document.getElementById('too-low');
 const maxGuessesMessage = document.getElementById('max-guesses');
 const numberOfGuessesMessage = document.getElementById('num-of-guesses');
 const correctMessage = document.getElementById('correct');
+const outOfRangeMessage = document.getElementById('out-of-range');
 
 let targetNumber;
-const attempts = 0;
-const maxNumberOfAttempts = 5;
-
-// Returns a random number from min (inclusive) to max (exclusive)
-// Usage:
-// > getRandomNumber(1, 50)
-// <- 32
-// > getRandomNumber(1, 50)
-// <- 11
+let attempts = 0;
+let maxNumberOfAttempts = 5;
+
 function getRandomNumber(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
 function checkGuess() {
-  // Get value from guess input element
   const guess = parseInt(guessInput.value, 10);
+  if (guess < 1 || guess > 100) {
+    //hideAllMessages(); not sure how it should work what if i type more than 100 after few tries what should show?
```

Comment:
It should show the red error message, but otherwise stay the same.  Also should not update 'you guessed'.

Reply:
Also should check for isNaN() since typing in case a non-numeric character is supplied.  I accidentally did.

Reply:
isNaN() will also catch a blank guess, which currently decrements the guess count.